client/my-sites/feature-upsell/feature.jsx

```diff
@@ -53,15 +53,16 @@ export default class Feature extends PureComponent {
 		}
 
 		return (
-			<PurchaseButton
+			<Button
```

Comment:
There are a few more occurrences of the `PurchaseButton`. Maybe we could just update those too, and remove it?

Reply:
The `PurchaseButton` should really be a private sub-component of the `PurchaseDetail`. And it's just a plain `Button` with additional `purchase-detail__button` class name, no additional logic there.

This change removes the only usage of `PurchaseButton` where there is no `PurchaseDetail` as a parent component.

I'd like if the direct imports of `components/purchase-detail/purchase-button` all disappeared in favor of using just `PurchaseDetail` with appropriate props (`description`, `body`, `buttonText`, ...). That shouldn't be hard to achieve. But it's better to do it in a separate PR. And it's not a terribly urgent task, to be honest.

The `feature-upsell` module is part of a feature that was never finished and is disabled behind a feature flag.